### ISSUE TITLE
Failed to load symbol sfWindow_setSize from shared library csfml-window-2.dl

### ISSUE THREAD

**Issue by nikibobi** [mdparker/Derelict3#74]
I build my program using Direlict sfml2, but when I run the executable I get this error:
`Failed to load symbol sfWindow_setSize from shared library csfml-window-2.dl`
Note that I have csfml dlls in the same folder.

**Comment by TheLionsRoar** [comment]
This error happens because the CSFML release candidate on the SFML website is quite outdated now, and Derelict SFML2 is made for updated versions of  CSFML. Rebuilding CSFML from the CSFML master repository solves this problem, though the SFML binding on this master is just enough out of date to create more similar errors. I've updated it on my own fork, which is awaiting a pull from Aldacron.

My build and repo I linked to in my reply to your post on SFML's forums should help you out: http://en.sfml-dev.org/forums/index.php?topic=9626.msg68426#msg68426
